### PR TITLE
Adding Bestiary Images

### PR DESCRIPTION
**Pull request by pcoffey42**
Adding images pulled from the book, and including monsters from T-Z.

### CODE REVIEW

File: ArcanaOfTheAncientsBestiary.json [hunk 1]
```diff
@@ -8532,6 +8532,1595 @@
...
+			"int": 5,
+			"wis": 13,
+			"cha": 6,
+			"senses": [
+			  "darkvision 60 ft."
+			],
+			"passive": 11,
+			"trait": [
+			  {
+				"name": "Languages—",
```
Comment (luathebee) [inline ArcanaOfTheAncientsBestiary.json]: fix language

File: ArcanaOfTheAncientsBestiary.json [hunk 2]
```diff
@@ -8532,6 +8532,1595 @@
...
+			  }
+			],
+			"hp": {
+			  "average": 114,
+			  "formula": "12d10 + 48"
+			},
+			"speed": {
+			  "walk": 30,
+			  "burrow": 30
+			},
```
Comment (luathebee) [inline ArcanaOfTheAncientsBestiary.json:8916]: Traits are really broken lol

File: ArcanaOfTheAncientsBestiary.json [hunk 3]
```diff
@@ -8532,6 +8532,1595 @@
...
+			  "charmed",
+			  "frightened",
+			  "grappled",
+			  "paralyzed",
+			  "petrified",
+			  "prone",
+			  "restrained",
+			  "stunned"
+			],
+			"trait": [
```
Comment (luathebee) [inline ArcanaOfTheAncientsBestiary.json:9207]: fix languages and cr

File: ArcanaOfTheAncientsBestiary.json [hunk 4]
```diff
@@ -8532,6 +8532,1595 @@
...
+			  "MW",
+			  "RCH",
+			  "RW",
+			  "THW"
+			],
+			"conditionInflict": [
+			  "incapacitated",
+			  "restrained"
+			],
+			"fluff": {
```
Comment (luathebee) [inline ArcanaOfTheAncientsBestiary.json:9523]: lots of entries here have a \ every now and then. Should it be like this?

File: ArcanaOfTheAncientsBestiary.json [hunk 5]
```diff
@@ -8532,6 +8532,1595 @@
...
+			},
+			"str": 3,
+			"dex": 10,
+			"con": 10,
+			"int": 8,
+			"wis": 8,
+			"cha": 5,
+			"skill": {
+			  "anyonewithproficiency": "+1"
+			},
```
Comment (luathebee) [inline ArcanaOfTheAncientsBestiary.json]: "anyonewithproficiency" is accpeted?

File: ArcanaOfTheAncientsBestiary.json [hunk 6]
```diff
@@ -8532,6 +8532,1595 @@
...
+			  "deafened",
+			  "exhaustion",
+			  "frightened",
+			  "paralyzed",
+			  "petrified",
+			  "poisoned"
+			],
+			"trait": [
+			  {
+				"name": "Languages—or one",
```
Comment (luathebee) [inline ArcanaOfTheAncientsBestiary.json]: fix languages and cr

**Review by luathebee — APPROVED** [review]

**Review by luathebee — APPROVED** [review]

**Comment by pcoffey42** [comment]
I went through and fixed the issues and imported into the 5e.Tools homebrew tool and foundry for testing. I checked each of the creatures and fixed errors as I found them. Most were in the T-Z monsters, but I saw and fixed a few minor issues in the earlier entries as well, if you want to go over them:

-The Mlox had a duplicate "type" entry (making it an "item" instead of a "construct").
-There were a couple of visual bugs with languages (Having "Language" set to "" causes a blank Languages entry, but if you delete the field altogether it properly displays null languages as "-", which seems to be 5e convention.)

**Review by luathebee — APPROVED** [review]
Looks great! Thanks for fixing and going over previous entries as well :D

**Pull request merged**